modules/migration-upgrading-migration-tool.adoc

```diff
@@ -60,17 +60,30 @@ $ sudo podman cp $(sudo podman create registry.redhat.io/rhcam-1-2/openshift-mig
 $ oc replace -f operator.yml
 ----
 
+ifeval::["{mtc-version}" == "1.2.4"]
+. If you are upgrading the {mtc-short} on an {product-title} 3.7 cluster, delete the Velero and Restic objects:
++
+----
+$ "oc delete --ignore-not-found=true deployment migration-controller migration-ui velero && oc delete --ignore-not-found=true daemonset restic
```

Comment:
Will you double-check the "? I can't tell if you have one too many or are missing one.

Reply:
Thanks for catching! It shouldn't be there.